docs/ai/conceptual/understanding-openai-functions.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Understanding OpenAI Function Calling"
+description: "Understand how function calling enables you to integrate external tools with your OpenAI application."
+author: haywoodsloan
+ms.topic: concept-article 
+ms.date: 04/26/2024
+
+#customer intent: As a .NET developer, I want to understand OpenAI function calling so that I can integrate external tools with AI completions in my .NET project.
+
+---
+
+# Understand OpenAI function calling
+
+Function calling is a feature of OpenAI models that supports describing functions and their arguments in a JSON format when making an completion request. The model won't invoke the function itself, but will return a JSON output describing what functions should be called and their arguments.
```

Comment:
```suggestion
Function calling is an OpenAI model feature that lets you describe functions and their arguments in prompts using JSON. Instead of invoking the function itself, the model returns a JSON output describing what functions should be called and their arguments.
```

Does the JSON output include the argument values?

Reply:
Yes, the JSON contains all that information. I reworded slightly to make that more clear.